File: SofiForce.Models/Models/ListModels/ClientGroupSubListModel.cs
using System;

namespace Models
{
    public class ClientGroupSubListModel
    {
        public int ClientGroupSubId { get; set; }
        public int ClientGroupId { get; set; }

        public string ClientGroupSubCode { get; set; }
        public string ClientGroupSubName { get; set; }
        public string ClientGroupName { get; set; }

        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/ClientGroupSubListModel.ts

export interface ClientGroupSubListModel {
    clientGroupSubId: number;
    clientGroupId: number | null;
    clientGroupSubCode: string;
    clientGroupSubName: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    icon: string;
    color: string;
    displayOrder: number | null;

}